src/source-pcap-file.c

```diff
@@ -136,6 +136,7 @@ void TmModuleDecodePcapFileRegister (void)
 
 void PcapFileGlobalInit()
 {
+    memset(&pcap_g, 0x00, sizeof(pcap_g));
```

Comment:
Good catch. There is also a call to memset on pcap_g in TmModuleReceivePcapFileRegister(). It would be better to remove it as it is now useless.